encode.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	vegeta "github.com/tsenart/vegeta/lib"
+)
+
+const (
+	encodingCSV  = "csv"
+	encodingGob  = "gob"
+	encodingJSON = "json"
+)
+
+type decoderFunc func(io.Reader) vegeta.Decoder
+type encoderFunc func(io.Writer) vegeta.Encoder
+
+func encodeCmd() command {
+	var (
+		fs     = flag.NewFlagSet("encode", flag.ExitOnError)
+		from   = fs.String("from", "gob", "Input decoding [csv, gob, json]")
+		to     = fs.String("to", "json", "Output encoding [csv, gob, json]")
+		output = fs.String("output", "", "Output file")
+	)
+
+	fs.Usage = func() {
+		fmt.Println("Usage: vegeta encode [flags] [<file>...]")
+		fs.PrintDefaults()
+	}
+
+	return command{
+		fs,
+		func(args []string) error {
+			fs.Parse(args)
```

Comment:
Error return value of `fs.Parse` is not checked

Reply:
Since we introduced @golangci, let's make it happy: `_ = fs.Parse()`